File: chapter01/akkademy-db-java/src/main/java/com/lp/akka/notes/messages/SetRequest.java
package com.lp.akka.notes.messages;

/**
 * @author li.pan
 * @version 1.0.0
 * @Description 消息体
 * @createTime 2020年12月16日 13:28:00
 */
public class SetRequest {
    private final String key;
    private final Object value;

    public SetRequest(String key, Object value) {
        this.key = key;
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public Object getValue() {
        return value;
    }
}

File: chapter04/akkademy-db-scala-4/src/main/scala/com/lp/akka/notes/messages/SetRequest.scala
package com.lp.akka.notes.messages

import akka.actor.ActorRef

/**
 * @title TODO
 * @author li.pan
 * @version 1.0.0
 * @createTime 2021年04月30日 14:14:00
 *             <p>
 *             $description
 *             </p>
 */
case class SetRequest(key: String, value: Object, sender: ActorRef = ActorRef.noSender) extends Request
